core/chains/evm/bulletprooftxmanager/bulletprooftxmanager_test.go

```diff
@@ -306,12 +308,81 @@ func TestBulletproofTxManager_CreateEthTransaction(t *testing.T) {
 		require.Error(t, err)
 		assert.Contains(t, err.Error(), fmt.Sprintf("cannot send transaction on chain ID 0; eth key with address %s is pegged to chain ID 1337", otherAddress.Hex()))
 	})
+
+	t.Run("simulate transmit checker", func(t *testing.T) {
```

Comment:
This is a bit misleading because it looks like its testing the checker but actually it's just testing the deserialisation. Maybe we can move this into `models_test.go` or something?

Reply:
It's also testing that the SQL reading and writing in bptxm.go is handling the data correctly, are you thinking move the tests as-is into models_test? Or only test the (de)serialization in models_test?